DTS-STN/canadian-dental-care-plan
# Adding URL and sub key for all API consumers; If not specified, default values for both are used

### Description
This change allows the flexibility to consume different Interop endpoints in our application.

If a specific API consumer's URL and/or subscription key are set, it will be used. Otherwise (the default scenario), the application will use the `INTEROP_API_BASE_URI` and `INTEROP_APPLICANT_API_SUBSCRIPTION_KEY` environment variables, which are mandatory.

### Checklist

### Test Instructions

- Run the application using the updated `.env.example` file as your `.env`. Ensure the application does not break in the areas where Interop would be called.
- Set an API's URL (eg. `INTEROP_CCT_API_BASE_URI=http://invalid.example.com`). Navigate to where that API is used (eg. `/en/letters`). The service call should use the new API URL (in this example scenario, the call will fail because `http://invalid.example.com` is not a valid endpoint)

### Additional Notes

- Node's `.env` file doesn't allow us to do something like:

```
INTEROP_API_BASE_URI=https://api.example.com
INTEROP_APPLICANT_API_BASE_URI=${INTEROP_API_BASE_URI}
```
...so I had to do the equivalent in the service files themselves

- Gitops project will be updated in a separate PR.
